Doc changes for FASC v2.7

manuwelakanade-GSLAB opened the pull request:
Doc changes for FASC v2.7

manuwelakanade-GSLAB commented:
Hi @JosephPJordan : Can you please review and merge this pull request. Thanks.

JosephPJordan commented on docs/faq.rst:
```diff
@@ -60,6 +60,14 @@ In order to recreate the deleted log files, disable and re-enable the applicatio
 
+------
+
+**Q. Why do I see the error "Error from BIG-IP: X-F5-Auth-Token does not exist" on performing a BIG-IP login from FASC app?**
```
... when performing a BIG-IP login..." is better.

JosephPJordan commented on docs/faq.rst:
```diff
@@ -60,6 +60,14 @@ In order to recreate the deleted log files, disable and re-enable the applicatio
+
+**Q. Why do I see the error "Error from BIG-IP: X-F5-Auth-Token does not exist" on performing a BIG-IP login from FASC app?**
+
+If the version of BIG-IP is changed and a re-login to the BIG-IP is attempted from the FASC app, this error may be seen.
```
...BIG-IP has changed, and you attempt to re-login to the BIG-IP from the FASC app, you may see this error.

JosephPJordan commented on docs/faq.rst:
```diff
@@ -60,6 +60,14 @@ In order to recreate the deleted log files, disable and re-enable the applicatio
+
+If the version of BIG-IP is changed and a re-login to the BIG-IP is attempted from the FASC app, this error may be seen.
+
+Workaround: Delete the BIG-IP from the FASC app UI and re-login to the BIG-IP again.
```
don't need both 're-' and 'again'.  So "...app UI and login to the BIG-IP again.

JosephPJordan commented on docs/faq.rst:
```diff
@@ -91,6 +101,40 @@ The F5 ACI ServiceCenter Visibility tables have an option to select the **Partit
+
+BIG-IP Endpoint Details section on the Visibility Dashboard may not display information due to the MAC address table getting flushed on the BIG-IP.
+
+Workaround: Send an ARP request to the host or check the connectivity with the host using ping command.
```
... using a **ping** command.  (or 'the ping command')

JosephPJordan commented on docs/faq.rst:
```diff
@@ -91,6 +101,40 @@ The F5 ACI ServiceCenter Visibility tables have an option to select the **Partit
+
+**Q. Why are the pool members displayed on Visibility VIP table, and the pool members displayed on the Visibility VIP dashboard not the same?**
+
+Visibility VIP table displays the pool members from a BIG-IP VIP, only if they are also present on the APIC. However, the Visibility dashboard shows all the pool members (and associated stats) that are present on the BIG-IP even if they may or may not be present on the APIC. Hence both the outputs may be different.
```
The visibility VIP table...

JosephPJordan commented on docs/faq.rst:
```diff
@@ -91,6 +101,40 @@ The F5 ACI ServiceCenter Visibility tables have an option to select the **Partit
+
+**Q. What does the field 'Route Domain' on the Visibility Dashboard indicate?**
+
+It displays the **default route domain** for the partition to which the Virtual Server (VIP) or Node belongs to.
```
... to which the Virtual Server (VIP) or Node belongs.  
<don't need 'to' at the end with "to which" earlier in the sentence>

JosephPJordan commented on docs/faq.rst:
```diff
@@ -91,6 +101,40 @@ The F5 ACI ServiceCenter Visibility tables have an option to select the **Partit
+
+------
+
+**Q. Why don't I see the scrollbar for 'View Logs' window on the Visibility Dashboard?**
```
...for the 'View Logs' window...

JosephPJordan commented on docs/faq.rst:
```diff
@@ -91,6 +101,40 @@ The F5 ACI ServiceCenter Visibility tables have an option to select the **Partit
+
+**Q. Why don't I see the scrollbar for 'View Logs' window on the Visibility Dashboard?**
+
+If you encounter this issue, please zoom out your web browser window by clicking the **Ctrl** key, and then press the **-** (i.e. minus key).
```
...this issue, use the 'zoom out' option on your web browser. For example, on Windows, hold the **Ctrl** key, and then click **-** (the dash/minus key).

JosephPJordan commented on docs/navigate.rst:
```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
+Configure Telemetry
+-------------------
+
+1. To view the Virtual Server statistics on the Visibility VIP Dashboard, Telemetry plugin version 1.17 or higher needs to be installed on the BIG-IP device.
```
...VIP Dashboard, you must install the Telemetry Streaming plugin version 1.17 or higher on the BIG-IP device.

JosephPJordan commented on docs/navigate.rst:
```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
+
+3. Configure a default pull consumer. For example: https://clouddocs.f5.com/products/extensions/f5-telemetry-streaming/latest/pull-consumers.html#pull
+
+4. Recommended pull consumer configuration: For a scaled environment it is highly recommended to retrieve just the Pool and Virtual Server stats from the BIG-IP. One way filter the VIP and Pool statistics is to create a default pull consumer configuration via  a POST request to: https://<BIG_IP>/mgmt/shared/telemetry/declare
```
For a scaled environment, we strongly recommend you retrieve just the Pool...

One way to filter the VIP... <missing the word 'to'>

JosephPJordan commented on docs/navigate.rst:
```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
+
+1. Click the Visibility tab, and then from the Table list, click the VIP table.
+
+2. From the Partition list, click the partition of interest.
```
...click the appropriate partition.

JosephPJordan commented on docs/navigate.rst:
```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
+
+5. Visibility Dashboard displays the information for the selected VIP including VIP name, service port, protocol, SNAT, route domain, iRules, default pool (and pool information such as load balancing and pool monitor)
+         
+         a. Dashboard displays a **Telemetry Consumer** drop-down list. Select the appropriate consumer from which the statistics need to be obtained. The Virtual pool statistics will be displayed on the dashboard including Bits, Packets, Connections and Requests. 
```
The dashboard displays...

JosephPJordan commented on docs/navigate.rst:
```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
+         
+            .. note::
+         
+               - Telemetry plugin needs to be installed on the BIG-IP to be able to view the Telemetry consumer list and stats from the telemetry consumer. Check the **Configure Telemetry** section for installation and configuration.
```
The Telemetry Streaming...

JosephPJordan commented on docs/navigate.rst:
```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
+            
+               - If there is only a single Telemetry pull consumer configured, then FASC will directly query that consumer and display the stats.
+            
+         b. **View Stats** - Dashboard displays a **View Stats** link besides the default pool. It displays total pool member count and the availability and enabled state of the pool  members using a pie chart.
```
The Dashboard... link next to the default pool <or beside, not besides>
It displays the total pool member count...

JosephPJordan commented on docs/navigate.rst:
```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
+            
+         b. **View Stats** - Dashboard displays a **View Stats** link besides the default pool. It displays total pool member count and the availability and enabled state of the pool  members using a pie chart.
+         
+         c. **View Logs** - Dashboard displays a **View Logs** link besides the VIP, which, on click will open a popup window and display all the logs for the selected Virtual Server. 
```
The Dashboard...  link next to the VIP, which, when clicked opens a popup window and displays...

JosephPJordan commented on docs/navigate.rst:
```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
+            
+               - It is possible to specify a UTC from and to Date/Time for the logs to be retrieved. The From/To time should be specified in UTC timezone. The response too will be in UTC.
+
+         d. **View Connections** - Dashboard will display a **View Connections** link which, on click will display all the active connections to that VIP.
```
The Dashboard displays a **View Connections** link, which, when clicked displays all the active...

JosephPJordan commented on docs/navigate.rst:
```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
+
+         d. **View Connections** - Dashboard will display a **View Connections** link which, on click will display all the active connections to that VIP.
+         
+         e. **APIC Endpoint Details** - Visibility dashboard displays APIC details for the VIP:
```
The Visibility dashboard...

JosephPJordan commented on docs/navigate.rst:
```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
+         e. **APIC Endpoint Details** - Visibility dashboard displays APIC details for the VIP:
+                                        1. MAC 2. EPG 3. Node 4. Interface 5. VLAN Encap
+                                        
+         f. **BIG-IP Endpoint Details** - Visibility dashboard displays BIG-IP details for the VIP:
```
The Visibility...

JosephPJordan commented on docs/navigate.rst:
```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
+
+1. Click the Visibility tab, and then from the Table list, click the Node table.
+
+2. From the Partition list, click the partition of interest.
```
... click the appropriate partition.

JosephPJordan commented on docs/navigate.rst:
```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
+   
+            - It is possible to directly click the **Visibility Dashboard** sub-tab and then select the Node, instead of a redirect from **Visibility Table**.
+      
+5. Visibility Dashboard displays the information for the selected VIP including Node name, route domain and monitor.
```
The Visibility Dashboard...

JosephPJordan commented on docs/navigate.rst:
```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
+      
+5. Visibility Dashboard displays the information for the selected VIP including Node name, route domain and monitor.
+
+         a. **View Logs** - Dashboard displays a **View Logs** link besides the Node, which, on click will open a popup and all the logs for the selected Node.
```
The Dashboard displays.....  link next to the Node, which, when clicked opens a popup...

JosephPJordan commented on docs/navigate.rst:
```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
+            
+               - It is possible to specify a UTC from and to Date/Time for the logs to be retrieved. The From/To time should be specified in UTC timezone. The response too will be in UTC.
+
+         b. **View Connections** - Dashboard displays a **View Connections** link which, on click will display all the active connections to that Node.
```
The Dashboard displays.... which, when clicked displays all the active...

JosephPJordan commented on docs/navigate.rst:
```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
+
+         b. **View Connections** - Dashboard displays a **View Connections** link which, on click will display all the active connections to that Node.
+
+         c. **APIC Endpoint Details** - Visibility dashboard displays APIC details for the Node:
```
The Visibility...

JosephPJordan commented on docs/navigate.rst:
```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
+         c. **APIC Endpoint Details** - Visibility dashboard displays APIC details for the Node:
+                                        1. MAC 2. EPG 3. Node 4. Interface 5. VLAN Encap 
+                                        
+         d. **BIG-IP Endpoint Details** - Visibility dashboard displays BIG-IP details for the Node:
```
The Visibility

JosephPJordan commented on docs/navigate.rst:
```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
+                                        1. MAC 2. EPG 3. Node 4. Interface 5. VLAN Encap 
+                                        
+         d. **BIG-IP Endpoint Details** - Visibility dashboard displays BIG-IP details for the Node:
+                                          1. MAC 2. VLAN 3. Interfaces (There is a **View Logs** link besides interfaces to view the interface logs) 4. Self IPs
```
"beside" or "next to"

JosephPJordan commented on docs/release-notes.rst:
```diff
@@ -1,3 +1,233 @@
+
+The user menu on the top-right corner of the F5 ACI ServiceCenter has an option for exporting F5 ACI ServiceCenter database (**Export/Import DB > Export DB**). This option does not work on APIC versions 5.0.1. This is working as expected on prior versions of the APIC and also on versions 5.0(2e) and higher. 
+
+**Workaround:** Backup the F5 ACI ServiceCenter database via the following steps:
```
...using the following steps:

JosephPJordan commented on docs/release-notes.rst:
```diff
@@ -1,3 +1,233 @@
+
+**Workaround:** Scroll slowly to prevent this issue. But if this issue is observed, the possible workarounds are:
+
+1. Switch tabs from Visibility tab to one of the other tabs and then switch back. 
```
..the Visibility tab

JosephPJordan commented on docs/release-notes.rst:
```diff
@@ -1,3 +1,233 @@
+
+**Application services declaration not deleted**
+
+If your AS3 declaration contains “optimisticLockKey” mentioned explicitly, the Application Services configuration may not be deleted completely, even after multiple attempts from the application UI. However, the configuration gets removed from the BIG-IP device.
```
However, the configuration is removed from the BIG-IP device.

JosephPJordan commented on docs/release-notes.rst:
```diff
@@ -1,3 +1,233 @@
+
+**Error on EPG mapping delete operation**
+
+When a dynamic endpoint mapping is added to an application using **Manage Endpoint Mappings**, the application gets created on the BIG-IP. If this mapping is deleted using the **RESET** button on **Manage Endpoint Mappings** form, users may encounter an error "The requested Pool Member (/Partition/App/Pool /NodePartition/NodeIP) was not found."
```
..., the application is created on the BIG-IP.

JosephPJordan commented on docs/release-notes.rst:
```diff
@@ -1,3 +1,233 @@
+
+**Pool members deleted or added directly to BIG-IP don't get updated automatically after clicking "Sync EPs".**
+
+1. If BIG-IP pool members automatically get added by the **Dynamic endpoint discovery** feature, but then few endpoints are deleted directly from the BIG-IP (i.e. out of band); these endpoints do not get created again on clicking **L4-L7 App Services --> Application Inventory --> Sync EPs** for that application. 
```
If BIG-IP pool members are automatically added by the...

JosephPJordan commented:
Hi @manuwelakanade-GSLAB, a few suggested changes.  Let me know if there's a better way we can do this (ie if i should just make changes directly to the files, etc).

manuwelakanade-GSLAB commented:
Hi @JosephPJordan : Thanks for the review. I have addressed all your comments and updated the pull request. Can you please merge this request if there aren't any further comments. Once this is merged, I will raise a pull request from staging to master branch. Thank you.

manuwelakanade-GSLAB commented:
Thanks @JosephPJordan
